source/js/buyers-guide/bg-main.js

```diff
@@ -172,6 +172,8 @@ let main = {
                 productID={parseInt(productID, 10)}
                 votes={votes}
                 whenLoaded={() => resolve()}
+                joinUsCSRF={csrfToken}
+                joinUsApiUrl={`${networkSiteURL}/api/campaign/signups/0/`}
```

Comment:
specifically, the template HTML we generate should inform what campaign signup this is for.

Reply:
This `<JoinUs />` component isn't in a template html. So it can't grab `{{ page.signup.id }} `. What would be another way I can get that value within jsx?

Reply:
I think this actually does need to be hardcoded in some way, as we're doing the same bit of unpleasantness for the [header/footer join components](https://github.com/mozilla/foundation.mozilla.org/blob/master/source/js/buyers-guide/bg-main.js#L210-L211). 

I'm going to file a follow up issue for us to fix this more broadly and switch to HTTP requests directly to Basket, as has been suggested by Pmac, rather than trying to shoehorn these through the signup endpoint/SQS.

That being said, it would be ideal if we could use the same less-than-ideal code linked above  in this case, so that when we do get to rip it out, it happens all at once.

Reply:
k, let's fix this in follow-up